merlin/models/tf/models/base.py

```diff
@@ -329,6 +337,27 @@ def compile(
             **kwargs,
         )
 
+    def _create_optimizer(self, optimizer):
+        if multi_gpu:
+            if isinstance(optimizer, merlin.models.tf.DistributedOptimizer):
+                # TODO
+                pass
+            if isinstance(optimizer, merlin.models.tf.MultiOptimizer):
+                # TODO
+                pass
+            else:
+                if isinstance(optimizer, str):
+                    optimizer = tf.keras.optimizers.get(optimizer)
+                scaled_lr = optimizer.learning_rate * hvd.size()
```

Comment:
I think we need to do this optional / adding a scaling parameter? In that way, it is enforced by our library. I dont know if we should do this?